io.github.rinigus.OSMScoutServer.json

```diff
@@ -0,0 +1,255 @@
+{
+    "app-id": "io.github.rinigus.OSMScoutServer",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "osmscout-server",
+    "finish-args": [
+        "--filesystem=host",
+        "--device=dri",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=network",
+        "--talk-name=org.freedesktop.DBus",
```

Comment:
Could you add a comment above this on how it is used at least.

Reply:
Comment added